flexmeasures/cli/data_add.py

```diff
@@ -1168,9 +1168,16 @@ def create_schedule(ctx):
     "--soc-at-start",
     "soc_at_start",
     type=QuantityField("%", validate=validate.Range(min=0, max=1)),
-    required=True,
+    required=False,
     help="State of charge (e.g 32.8%, or 0.328) at the start of the schedule.",
 )
+@click.option(
```

Comment:
Is the plan to let `soc` deprecate `soc-at-start`?

Having both seems confusing to me.

Reply:
Felix suggested to have a different field for `soc` in the flex model.If both are passed then validation will fail.

Reply:
@Flix6x can you comment on the difference of these fields, and how to tell the user? They both seem to be about the same thing, and I believe it would be good to reduce to one.
In any case, the help text should be better.

Reply:
> Is the plan to let soc deprecate soc-at-start?

Yes, but not yet in this PR.

I consider this PR to lead us to an in-between state where we don't yet *need* to document the new field, as it could indeed lead to confusion. After this PR, it should be possible to let storage schedulers save their SoC schedules to an SoC sensor, by passing `{"sensor": <id>}` to the undocumented `soc` field of the flex-model. Then, as a follow-up PR, we can use the same sensor to fetch the starting SoC from, add documentation for the new field, and think about deprecating the `soc-at-start` field.